app_dart/micro_services/results_pipeline/lib/common/auth_utils.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
```

Comment:
Seems like this file should go to common libraries.

Reply:
ditto comment as above

Reply:
One issue I faced while making the scheduler microservice is how appengine handles dart packages. You can't rely on the local references or git, and you'll need to land your package change, publish it, then land your microservice change. I'd like to see if we can find a way around it and get the local paths working here.